src/common/StatsCard/StatsCard.js

```diff
@@ -40,20 +40,24 @@ StatsCard.Header = ({ children, title }) => {
 }
 
 StatsCard.Body = ({ children }) => {
-  return <div className="stats-card__row stats-card__full-row">{children}</div>
+  return <div className="stats-card__row">{children}</div>
 }
 
 StatsCard.Footer = ({ children }) => {
   return <div className="stats-card__row">{children}</div>
 }
 
+StatsCard.Row = ({ children }) => {
+  return <div className="stats-card__row">{children}</div>
+}
+
```

Comment:
`StatsCard.Row` and `StatsCard.Body` and `statsCard.Footer` have the same class `stats-card__row`. That's doesn't seem right. In this case there is no sense to have separate blocks that are identical.